build_debian.sh

```diff
@@ -475,6 +475,9 @@ fi
 ## Disable kexec supported reboot which was installed by default
 sudo sed -i 's/LOAD_KEXEC=true/LOAD_KEXEC=false/' $FILESYSTEM_ROOT/etc/default/kexec
 
+# Ensure that 'logrotate-config.service' is set as a dependency to start before 'logrotate.service'.
```

Comment:
We'll also need to modify `files/image_config/logrotate/logrotate-config.service` to add `RemainAfterExit=yes` to it, so the service is treated as running after it script executes.